modules/tensor_mechanics/include/auxkernels/TensorElasticEnergyAux.h

```diff
@@ -26,7 +26,8 @@ class TensorElasticEnergyAux : public AuxKernel
 protected:
   virtual Real computeValue();
 
-private:
+  std::string _base_name;
+
```

Comment:
This looks great!

Just kidding, this message is only for MOOSE training demonstration.